arroyo-api/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "arroyo-api"
 version = "0.3.0"
 edition = "2021"
+license = "MIT"
```

Comment:
@mwylde utoipa reads this to include it in the openapi spec. I see there's both Apache and MIT licenses in the repo. Do you know which one applies to use of the API?

Reply:
Arroyo is dual-licensed under Apache and MIT, so the API is both.